kemeny_sourcemash.tex

```diff
@@ -35,7 +35,9 @@
 \pagebreak
 \section{Project Overview}
 
-Sourcemash 
+Sourcemash is a web-based application that categorizes news articles to help users read their RSS feeds faster. Sourcemash achieves this goal is three steps. First, users select their sources from the Browse page, which offers a pre-seeded list of popular feeds and a form allowing manual input of new RSS feeds; Sourcemash is designed to categorize your news, not someone else's. Second, Sourcemash's categorization engine analyzes each article in the user's subscribed feeds to find the categories that best match; our algorithm is constantly working in the background, so users can find articles in the topics they're looking for faster. Finally, Sourcemash offers the ability to read thew news by both category and feed, offering users the ability to integrate their news reading preferences into the application experience. 
+
+Sourcemash targets users of RSS feed aggregator subscriptions -- popular web-app Feedly currently has a market size of 15 million users (http://googlecloudplatform.blogspot.com/2014/04/feedly-delivers-seamless-experience-to-15-million-users-with-app-engine-and-cloud-storage.html) with 50,000 paying for premium services -- and aims to enhance the news reading experience by eliminating the article redundancy inherent in subscribing to similar news sources and offering a category-driven perspective to the news that reduces the time required to parse related content. 
```

Comment:
Move Feedly out of the opening sentence. Maybe add a list of competitive RSS feed readers at the end and say that we're the only ones with automatic categorization.